index.js

```diff
@@ -32,12 +32,21 @@ String.prototype.unicodeToMerica = function () {
     return cleanseString(this); 
 }
 
-/// We want ENTERPRISE LEVEL runtime configuration
-const config = {
-    TheRealests: process.env.THE_REALEST_MFERS?.split(';') ??  [ 'kerouac5' ],
-    
-    OneBlockedPercent: process.env.ONE_BLOCKED_PERCENT ?? 5
-};
+
+
+/**
+ * Gets the config but cleans out any values that should be secret.
+ */
+getCleansedConfig = () => ({ ... config, Token: undefined });
+
+/**
+ * Determines if the user is one of the realest mother fuckers there is.
+ * @param {string } username the query
+ * @returns true if user is one of the realest;
+ */
+function isRealest(username) {
+    return config.TheRealests.filter((k) => k.toLocaleLowerCase() === username);
```

Comment:
so the bug @bran-solo was this used to be `=== k` as opposed to `=== userName` but I hope encapsulating this into a function makes things easier to read/